### DISCUSSION TITLE
web-app 에 관한 질문

### DISCUSSION THREAD

**Discussion by parkjh97** [dineug/erd-editor#346] [Q&A]
안녕하세요 
제공하는 api 문서를 읽고 질문 드립니다. 
현재 제공하고 있는 공동편집, 원격저장 에관하여 읽고 web app 에서 놀라운 기능들을 봤습니다.

혹시 web-app 에 new Schema 와 같은 기능들을 사용 할수 있는 방법 이있을까요..?

**Comment by dineug** [comment]
web-app에 구현된 기능을 라이브러리 기능 제공을 말씀하시는거면
라이브러리가 아니기때문에 외부에서 사용할 수 없습니다.
web-app 구현을 참고하여 직접구축을 추천드려요

**Reply by parkjh97** [reply]
답변 감사합니다 !
